src/common/components/Modal/ConfirmModal/ConfirmModal.tsx

```diff
@@ -0,0 +1,31 @@
+import classNames from "classnames/bind";
+import styles from "@/common/components/Modal/ConfirmModal/confirmModal.module.scss";
+import { useRef } from "react";
+import Modal from "@/common/components/Modal/Modal";
+import useOutSideClick from "@/common/components/Modal/hooks/useOutsideClick";
+import Button from "@/common/components/Button/Button";
+
+const cn = classNames.bind(styles);
+
+interface ConfirmModalProps {
+  message: string;
+  className: string;
+  onClick: () => void;
+}
+
+export default function ConfirmModal({ message, className, onClick }: ConfirmModalProps) {
+  const modalRef = useRef<HTMLDivElement>(null);
+
+  useOutSideClick(modalRef, onClick);
```

Comment:
outside는 한단어로 표현하는 편입니다~
```suggestion
  useOutsideClick(modalRef, onClick);
```